fixtures/report_test.csv

```diff
@@ -0,0 +1,6 @@
+﻿original_path,normalized_path,predicted_class,predicted_conf
+http://localhost:8081/ncc/Cm912_1945u1_sheet1.jpg,/ml-repo-preingest-processing/fixtures/normalized_images/ncc/Cm912_1945u1_sheet1.jpg,1,0.523

```

Comment:
For the sake of the data being realistic, the `original_path` field is not going to be a http URL, it would be a file path, so you could just replace it with `/path/to/ncc/Cm912_1945u1_sheet1.jpg`